tests/integration/targets/user_grants/tasks/main.yaml

```diff
@@ -71,7 +71,7 @@
     - name: Assert grants updated
       assert:
         that:
-          - no_changes.changed == False
```

Comment:
Is this needed for skipping the long tests?

Reply:
This is to address an unrelated test failure